deploy/infra/aws/tf-deploy-infra/vars.tf

```diff
@@ -0,0 +1,74 @@
+variable "cp_account_id" {
```

Comment:
please try to be more specific for each of the parameter. See my other comments for this file for the examples.